### ISSUE TITLE
Setting c# variable from python during trainig

### ISSUE THREAD

**Issue by fashrista** [SciSharp/Keras.NET#214]
I am trying to visualize the training process in a c# app but am having trouble accessing training metrics (loss,val_loss, accuracy, mse) during model training. Creating a callback method in Python.py file as suggested in #13  does not help because I cannot access that data from c# application.
Has anybody seen an example of this, or has anybody a suggestion? The only idea I have is to write a series of files to HDD in .py callbacks, which with some delay can be then read by the c# app ... far from ideal.

Access this kind of training metrics from c#: 
![slika](https://user-images.githubusercontent.com/37577804/144727130-fed7f4ee-d9a4-4fd7-a0b9-1390114294e1.png)

**Comment by fpisarello** [comment]
Hello, I need this feature too, but i don't know if **pythondotnet** do that.
